src/transform/WidenImage.js

```diff
@@ -25,6 +25,9 @@ const widenAncestors = el => {
     if (parentElement.style.float) {
       parentElement.style.float = 'none'
     }
+    if (parentElement.style.display) {
+      parentElement.style.display = 'block'
```

Comment:
What do you think about using `inline-block` here? I was thinking it might be safer than `block`. For example, if the image isn't set to 100% width for whatever reason, `inline-block` seems more likely to work than `block`.

Reply:
Inline block may make tablet layouts get a bit funky...

Reply:
Yeah, I'm trying to get the old Nexus 9 charged up to test this myself :)

Reply:
I did some manual testing and this seems fine in practice.  In principle, though, wouldn't the effect of this be to allow other images to appear beside widened images?  Is that something we want?